schema/record-package-schema.json

```
@@ -132,7 +132,7 @@
         },
         "releases": {
           "title": "Releases",
-          "description": "All the existing OCDS releases related to the contracting process. The releases MUST be sorted into date order in the array, from oldest (at position 0) to newest (last).",
+          "description": "A list of all the OCDS releases related to the contracting process at the time of the record's publication. The releases MUST be sorted into date order in the array, from oldest (at position 0) to newest (last).",
```

We try to avoid information about data representation (data types like list or array) in the description.

I also simplified the second clause, as "position 0" might be confusing, and I also do not know how a "from ... to ..." construction could be interpreted in any other way than "from (first) to (last)".

Stylistically, we also do not capitalize normative keywords elsewhere.

```suggestion
          "description": "All the releases about the contracting process at the time of the record's publication. The releases must be sorted in date order, from oldest to newest.",
```